namer/consul/src/test/scala/io/buoyant/namer/consul/ConsulNamerTest.scala

```diff
@@ -573,72 +573,10 @@ class ConsulNamerTest extends FunSuite with Awaits {
     @volatile var state: Activity.State[NameTree[Name]] = Activity.Pending
     namer.lookup(Path.read("/dc1/servicename/residual")).states respond { state = _ }
 
-    assert(state == Activity.Ok(NameTree.Neg))
-  }
-
-  test("Namer state returns Neg then Bound then Neg when a datacenter becomes available and then becomes unavailable") {
```

Comment:
We no longer need this test since this tests the behavior of having a datacenter to become unavailable. we don't expect this to happen at runtime.

Reply:
Are you sure? I don't see any reason why a DC couldn't become unavailable.  I would expect in that case that the vaddr would should go from having a value to being neg.

Reply:
Looking through the comments in #2145, the user mentioned that 
>  A DC doesn't usually cease to exist during runtime
Based on that, I felt that the test wasn't necessary since this issue only happens on the initial call to resolve the logical. But since this can be considered edge case, it might be helpful to have this test.